renderer/src/pages/settings/Settings.tsx

```diff
@@ -1,11 +1,19 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
```

Comment:
what is this used for?

Reply:
To disable the warning about using // inside text. But if we use html entities instead, I can remove it